packages/api/src/modules/payment/payment-service.ts

```diff
@@ -0,0 +1,16 @@
+import { PaymentProvider } from "@dotkomonline/types"
+import { readableStripeAccounts } from "./../../lib/stripe"
+
+export interface PaymentService {
+  getPaymentProviders(): (PaymentProvider & { paymentAlias: string })[]
+}
+
+export class PaymentServiceImpl implements PaymentService {
```

Comment:
This class seems a bit small? mayhaps it should be merged with another service

Reply:
Agreed. Everything named transaction is now named payment, and the old transaction service and router was merged with the existing ones. The product router is still accessible as a subrouter through payment router.